libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -92,6 +92,9 @@ void AP_AHRS_NavEKF::update(void)
     update_SITL();
 #endif
 
+    // update AOA and SSA
```

Comment:
is this useful for non-fly-forward vehicles? for example, copters and rovers?
Perhaps wrap it in a check for get_fly_forward() ?